### DISCUSSION TITLE
Fonts in Fluent UI React Components v9

### DISCUSSION THREAD

**Discussion by osnoser1** [microsoft/fluentui#26576] [Q&A] [answered]
Hi, I'm looking in the documentation examples and I can't see if there is a standard way to load the fonts (segoe ui) in a new project with Fluent UI React Components v9, I mean, I haven't been able to find an example project where the *.woff files are being downloaded.

I would like to know if there is a documented way to do it, or should I implement it in my own way?.

Thank you very much in advance!

**Comment by micahgodbolt** [comment, marked as answer]
We don't have any prescribed way to load fonts in v9. Check out the asset license guidance https://github.com/microsoft/fluentui#licenses
